data-science-onramp/data-ingestion/ingestion.py

```diff
@@ -35,19 +35,27 @@
 CITIBIKE_TABLE_NAME = "RAW_DATA"
 EXTERNAL_TABLES = {
     "gas_prices": {
-        "url": "https://data.ny.gov/api/views/wuxr-ni2i/rows.csv",
+        "url": "https://data.ny.gov/api/views/nqur-w4p7/rows.csv",
         "schema": StructType(
```

Comment:
nit: to save indentation, would this work?

```
SCHEMA_DATA = StructType( ... )
EXTERNAL_TABLES = {
    "gas_prices": {
        "url": ...,
        "schema": SCHEMA_DATA,
    },
}
```

If not, fine with what we have at the moment.

Reply:
I just tried it out locally and honestly, it did not make too much of a visual difference to me - given that it works, I'm going to take the option to keep it as is!